packages/fxa-auth-server/lib/payments/paypal/error.ts

```diff
@@ -0,0 +1,12 @@
+export class PaypalFailure extends Error {}
```

Comment:
A type alias for future use because you are going to add more `PaypalFailure`s?

Reply:
Yep, and a single inheritance point as well.